build.sh

```diff
@@ -25,7 +25,13 @@ __scriptpath=$(cd "$(dirname "$0")"; pwd -P)
 
 "$__scriptpath/build-native.sh" $*
 if [ $? -ne 0 ];then
-   exit 1
+    exit 1
+fi
+
+# Building CoreFx.Tools before calling build-managed.sh to ensure our tools are available before running the main build.
+"$__scriptpath/Tools/msbuild.sh" "$__scriptpath/src/Tools/CoreFx.Tools/CoreFx.Tools.csproj" /v:m /m
```

Comment:
Actually, this was a misunderstanding on my part. There was a configuration option in MSBuild.runtimeconfig.json (which we override) that gave us the old behavior back. I accidentally deleted that file. After un-deleting it, we don't need this hacky stuff.